spring/1.3/modules/ROOT/pages/spring-module-troubleshooting.adoc

```diff
@@ -0,0 +1,47 @@
+= Troubleshooting Spring Module - Mule 4
+
+To troubleshoot Spring module, become familiar with the information about enabling verbose logging and interpreting commonly thrown messages.
+
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with Spring module is failing, temporarily enable verbose logging for the module.
+
+Remember to always disable the enhanced verbosity after troubleshooting, because it affects your Mule application performance.
+
+To enable verbose logging in the configuration file:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. Add an `<AsyncLogger>` tag inside the `<Loggers>` tag:
++
+[source,xml,linenums]
+----
+			<Loggers>
+				...
+				<AsyncLogger name="org.mule.extension.spring" level="DEBUG"/>
+				...
+			</Loggers>
+----
+[start=6]
+. Save your application changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+
+
+== Understand Common Throws
+
+Here is a list of common throw messages and how to interpret them:
+
+* MULE:NOT_PERMITTED
+
+ This error occurs when there is a security restriction enforced by a filter. For example, using the authorization filter of the Spring module.
+
+* MULE:SERVER_SECURITY
+
+ A security error enforced by the Mule runtime engine (Mule).
```

Comment:
It would be good if you can add more information about this error - maybe give examples of when it can occur. Also, the leading words should be parallel to the other ones:

Occurs when there is a security error....